amundsenatlastypes/schema/01_4_report.json

```diff
@@ -0,0 +1,57 @@
+{
+  "entityDefs": [
+    {
+      "superTypes": [
+        "Referenceable"
+      ],
+      "name": "Report",
+      "description": "External data reports ( url's) associated with table",
```

Comment:
doesn't have to be associated with table

Reply:
fixed in 984cbc1